shared/components/splash-example.jsx

```diff
@@ -59,7 +59,7 @@ export default React.createClass({
                             # Photos:
                         </span>
 
-                        <select
+                        <select className="num-photos-select"
```

Comment:
Okay sounds good. Having it this way ends up testing whether or not we probably setup the event handling in the React component as well. Whereas, manipulating this React component directly would not verify that it correctly responds to DOM events from users interacting with the controls.